tests/whitebox/integration/logical/test_destroy.py

```diff
@@ -108,3 +109,40 @@ def testDestroy(self):
         """
         command_line = self._MENU + [self._POOLNAME] + self._VOLNAMES
         RUNNER(command_line)
+
+
+class Destroy4TestCase(SimTestCase):
+    """
+    Test destroying multiple volumes, of which one does not exist.
+    """
+
+    _MENU = ["--propagate", "filesystem", "destroy"]
+    _POOLNAME = "deadpool"
+    _VOLNAMES = ["oubliette", "mnemosyne"]
+
+    def setUp(self):
+        """
+        Start the stratisd daemon with the simulator.
+        """
+        super().setUp()
+        command_line = ["pool", "create", self._POOLNAME] + _DEVICE_STRATEGY()
+        RUNNER(command_line)
+
+        command_line = ["filesystem", "create", self._POOLNAME] + self._VOLNAMES[0:1]
```

Comment:
Can you leave out these two lines and ....

Reply:
I think you mean Lines 137-138.